### PR TITLE
Correct indentation, CamelCase function and content tag

### PR DESCRIPTION
**Pull request by eloypastor**

### CODE REVIEW

File: bin/new_element.dart [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+
+void _createBoilerPlate(
+    String element, String directory, int distanceToPackageRoot) {
+
+  var capitalizedName = _toCamelCase(element);
+  var underscoreName = element.replaceAll('-', '_');
+
+  String html = '''
+<dom-module id="$element">
+    <template>
```
Comment (jakemac53) [inline bin/new_element.dart]: Sorry to keep nitpicking this but lets stick with either 4 or 2 space indent throughout. I don't have an opinion really on which, but it should be consistent. (template tag here is indented 4, rest is indented 2)

**Comment by jakemac53** [comment]
Ok, one last thing. Do you mind squashing/renaming these commits? Otherwise lgtm

**Comment by eloypastor** [comment]
Sorry, I don´t understand you

**Comment by jakemac53** [comment]
Currently there are two commits, one with the title "Correct indentation, CamelCase function and content tag", and one with the title "correct indentation". In the previous pull request (https://github.com/dart-lang/polymer-dart/pull/657/commits) the first commit was titled "Create Polymer 1.0 elements automatically" which was more descriptive of the actual issue.

It would be nice if you could at least rename the first commit here to be the same.

**Comment by jakemac53** [comment]
oh whoops didn't see that this had been updated, merging now thanks!

**Pull request merged**